src/v1/config.ts

```diff
@@ -26,6 +26,14 @@ import * as path from 'path';
 import * as firebase from 'firebase-admin';
 
 export function config(): config.Config {
+  // K_CONFIGURATION is only set in GCFv2
+  if (process.env.K_CONFIGURATION) {
+    throw new Error(
+      'RuntimeConfig is no longer available in Google Cloud Functions v2. ' +
```

Comment:
```suggestion
      'functions.config() is no longer available in Cloud Functions for Firebase v2. ' +
```

we never actually called it  "Runtime Config" other than the filename, and only Firebase has it at all